back/spec/models/custom_field_spec.rb

```diff
@@ -81,6 +81,45 @@ def visit_page(_field)
     end
   end
 
+  describe 'title_multiloc validation' do
+    let(:form) { create :custom_form }
+
+    it 'happens when the field is not a page' do
```

Comment:
I don't see a case for a valid title here, but on the other hand, I'm not sure if we should test our validations for all possible situations.

Reply:
I will add it.